apps/parsley/vite.config.ts

```diff
@@ -12,114 +12,119 @@ import path from "path";
 import { generateBaseHTTPSViteServerConfig } from "@evg-ui/vite-utils";
```

Comment:
I think this might break the staging / beta deploys

I just tried in a [patch](https://spruce.mongodb.com/task/evergreen_ui_deploy_deploy_spruce_staging_patch_97c982fb52d4f4a2e1b86b1332f2bc4b9019ef76_67ddd40a24d8d700074f34b0_25_03_21_21_03_18/logs?execution=0) on this PR (no changes, clean) and it failed for me

Reply:
Great catch! Updated the script to disable script mode when running yarn build
https://spruce.mongodb.com/version/67ddd6bc8b22cb0007f95dfe/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC